style.css

```diff
@@ -36,11 +36,16 @@ p {
   display: none;
```

Comment:
- [ ] If you check your website there is some overflow in the x-axis so I suggested you go to your body selector and put the `overflow-x: hidden;` property will solve your issue check below :point_down: my suggestions and screenshot :slightly_smiling_face: 

suggestion: 
```css
body {
  background-color: #e5e5e5;
  margin: 0;
  overflow-x: hidden;
}
```

**Your design:**
![screencapture-127-0-0-1-5502-2023-02-23-22_10_07](https://user-images.githubusercontent.com/45226848/220986295-68c53b99-b728-41c4-a77e-0bc3ce6b2e0d.png)